sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -4118,6 +4118,26 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
     }
   }
 
+  test("try_cast") {
```

Comment:
Could we move this test into `SQLQueryTestSuite`?

Reply:
make sense.